cli/client/user.go

```diff
@@ -2,15 +2,16 @@ package client
 
 import (
 	"encoding/json"
-	"net/url"
 
 	"github.com/sensu/sensu-go/types"
 )
 
+var usersPath = createBasePath(coreAPIGroup, coreAPIVersion, "rbac", "users")
```

Comment:
I think this should be `createBasePath(coreAPIGroup, coreAPIVersion, "users")`, ie: drop the `rbac`.